src/main/java/webApp/webController.java

```diff
@@ -146,4 +146,15 @@ public Response addAnswers(@RequestBody SurveyMessage surveyMessage) {
         }
         return new Response(null, "error", "Survey \""+ surveyMessage.getId() +"\" not available");
     }
+
+    @DeleteMapping(value = "/survey/{id}", produces = "application/json")
+    @ResponseBody
+    public Response deleteSurvey(@PathVariable int id) {
+        Optional<Survey> survey = repo.findById(id);
+        if (survey.isPresent()) {
+            repo.deleteById(id);
+            return new Response(id, "ok", "Survey deleted");
```

Comment:
Can we append the id to the response? So like "Survey 1 deleted", just so we get confirmation of the survey we deleted.